ttnn/cpp/ttnn/operations/data_movement/reshape_view/reshape_pybind.cpp

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "reshape_pybind.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+#include "ttnn/operations/data_movement/reshape_view/reshape.hpp"
+#include "ttnn/types.hpp"
+
+namespace ttnn::operations::data_movement {
+
+namespace detail {
+
+template <typename data_movement_operation_t>
+void bind_reshape_view(pybind11::module& module, const data_movement_operation_t& operation, const char* doc) {
+    bind_registered_operation(
+        module,
+        operation,
+        doc,
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const ttnn::Shape& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, new_shape);
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const std::vector<int32_t>& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, new_shape);
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            }
+        );
+}
+
+}  // namespace detail
+
+
+void py_bind_reshape_view(pybind11::module& module) {
+    detail::bind_reshape_view(
+        module,
+        ttnn::reshape,
+
+        R"doc(
+
+        Args:
+            * :attr:`input_tensor`: Input Tensor.
```

Comment:
please get docs guide from @yan-zaretskiy , I think :attr: is not needed anymore

Reply:
ah true, sorry forgot to push that commit